.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+name: Build
+# Tests runs different tests (test_abci_apps, test_abci_cli, test_apps)
+# This workflow runs on every push to master or release branch and  every pull requests
+# All jobs will pass without running if no *{.go, .mod, .sum} files have been modified
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - release/**
```

Comment:
What creates these? I see we have them for releases up through v0.34.0, then one for v0.34.13, but otherwise none since that. Are we missing a release instruction, or are they possibly obsolete?

Reply:
they might be obsolete. presumably our branching strategy changed?